packages/strapi-provider-upload-cloudinary/lib/index.js

```diff
@@ -19,7 +19,8 @@ module.exports = {
             { resource_type: 'auto', public_id: file.hash, ...customConfig },
             (err, image) => {
               if (err) {
-                return reject(err);
+                console.log(`Error uploading to cloudinary: ${err.message}`);
```

Comment:
here you will need to use strapi.log.error()

Reply:
Updated